src/NzbDrone.Core/DecisionEngine/DownloadDecisionPriorizationService.cs

```diff
@@ -13,18 +15,23 @@ public interface IPrioritizeDownloadDecision
     public class DownloadDecisionPriorizationService : IPrioritizeDownloadDecision
     {
         private readonly IDelayProfileService _delayProfileService;
+        private readonly IIndexerFactory _indexerFactory;
+        private readonly Logger _logger;
 
-        public DownloadDecisionPriorizationService(IDelayProfileService delayProfileService)
+        public DownloadDecisionPriorizationService(IDelayProfileService delayProfileService, IIndexerFactory indexerFactory, Logger logger)
         {
             _delayProfileService = delayProfileService;
+            _indexerFactory = indexerFactory;
+            _logger = logger;
         }
 
         public List<DownloadDecision> PrioritizeDecisions(List<DownloadDecision> decisions)
         {
+            var indexers = _indexerFactory.All();
             return decisions.Where(c => c.RemoteEpisode.Series != null)
                             .GroupBy(c => c.RemoteEpisode.Series.Id, (seriesId, downloadDecisions) =>
                                 {
-                                    return downloadDecisions.OrderByDescending(decision => decision, new DownloadDecisionComparer(_delayProfileService));
+                                    return downloadDecisions.OrderByDescending(decision => decision, new DownloadDecisionComparer(_delayProfileService, _logger, indexers));
```

Comment:
A logger should not be passed in, they are class specific. Take a look at how `Parser` gets a logger.